stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -125,6 +125,9 @@ public enum _RuntimeConfig {
   /// needed. Otherwise, The entire GPU memory region is pre-allocated.
   static public var gpuMemoryAllowGrowth = true
 
+  /// The number of CPU devices.
+  static public var numCPUDevices: UInt32 = 1
```

Comment:
Swift conventions perfer "xxxCount" instead of "numXXX".
```suggestion
  static public var cpuDeviceCount: UInt32 = 1
```

Reply:
Done